css/style.css

```diff
@@ -752,6 +753,104 @@ ul.social-link .icon {
   margin-right: 140px;
 }
 
+.project-modal {
+  background: #fff;
+  border: 1px solid hsl(223, 12%, 89%);
+  width: 90%;
+  max-width: 90%;
+  height: 98vh;
+  padding: 0 10px;
+  margin: 12px auto;
+}
+
+.modal-container {
+  position: fixed;
+  inset: 0 0;
+  background-color: rgba(0, 0, 0, 0.8);
+  width: 100%;
+  height: 100%;
+  display: none;
+}
```

Comment:
- [ ] You did a good job on the functionalities but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design as much as possible. As requested by the previous reviewer Please increase the top margin and make sure it matches the Figma design. Please add a blur effect in the background.
  - For blur you can use effect `backdrop-filter: blur(5px);`, use px according to your design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/168036303-2993d099-d192-4090-a4f5-7bfbe483de96.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/168036348-47b9ae86-8ffb-449b-8c2a-8a3a9dd85d61.png)